packages/falso/src/lib/account.ts

```diff
@@ -17,5 +17,11 @@ import { fake, FakeOptions, getRandomInRange } from './core/core';
 export function randAccount<Options extends FakeOptions = never>(
   options?: Options
 ) {
-  return fake(() => getRandomInRange({ max: 99999999 }).toString(), options);
+  const factory: () => string = () => {
+    return '#########'.replace(/#/g, () => {
```

Comment:
I think we have a function for this functionality

Reply:
Ah ok, i'll have a look and also change the card numbers function to use it. Do you know what it's called?

Reply:
@NetanelBasal I've had a look and it seems like we only do it in 2 places; `randAccount` and `randCreditCardNumber`. Both do it manually.